main.py

```diff
@@ -0,0 +1,11 @@
+from cycle import *
+
+if __name__ == "__main__":
+    firstCycle = Cycle(1, 40, 120, "Jarik", 14, "blue", False)
+    secondCycle = Cycle(24, 20, 100, "Desna",  14, "blue",)
+    thirdCycle = Cycle(16, 60, 220, "Ukraina")
+
+    print("first cycle: ", firstCycle.__str__())
```

Comment:
You don't need to call `__str__()`explicitly; `print` invokes `str(x)` implicitly.

```suggestion
    print("first cycle: ", firstCycle)
```